### PR TITLE
Add more Composer commands to the appendix

### PR DESCRIPTION
**Pull request by heliosfa**
Add all of the Composer commands to address https://github.com/POETSII/Orchestrator/issues/258

### CODE REVIEW

**Comment by mvousden** [comment]
I generally prefer alphabetical order so that it's easier to find a given command, but I can see why you've done it that way.

File: source/user_guide.md [hunk 1]
```diff
@@ -666,35 +666,61 @@ configured staging directory (`Output/Composer` in the default configuration).
...
+ - `compose /clean`: Given a placed application graph instance (or multiple),
+   clear any compiled binaries composed for that application.
+
  - `compose /bypass`: Bypasses most of the compose process provided that the
    compiled binaries for the application already exist, allowing the operator
    to reuse binaries from a previous run or to use binaries compiled elsewhere. 
    The loaded application must be identical in terms of definition and placement 
    for this to work - there are no checks beyond binary existance.
 
+ - `compose /reset`: Deletes the Composer, clearing all data, and creates a new one.
```
Comment (mvousden) [inline source/user_guide.md:694]: "creating a new one" avoids a tense change.

File: source/user_guide.md [hunk 2]
```diff
@@ -666,35 +666,61 @@ configured staging directory (`Output/Composer` in the default configuration).
...
+
+ - `compose /dump`: Dumps diagnostic data regardign the Composer to the microlog.
```
Comment (mvousden) [inline source/user_guide.md]: "regarding"

File: source/user_guide.md [hunk 3]
```diff
@@ -666,35 +666,61 @@ configured staging directory (`Output/Composer` in the default configuration).
...
+ 
+ Additionally, the behaviour of the generated Softswitch can be controlled by setting
+ Composer parameters:
+
  - `compose /args`: Allows the operator to pass additional arguments to the
    compiler(s) used to build the application binaries.
 
- - `compose /dump`: Dumps diagnostic data regardign the Composer to the microlog.
+ - `compose /inst`: Indicates that the Softswitch should be compiled with
```
Comment (mvousden) [inline source/user_guide.md]: Probably should be "softswitches"? Not sure. (same for all that follow)

File: source/user_guide.md [hunk 4]
```diff
@@ -666,35 +666,61 @@ configured staging directory (`Output/Composer` in the default configuration).
...
+   non-buffering mode (default).
+
+ - `compose /logh`: Indicates that the Softswitch should be compiled with 
+   the specified log handler. Options are `none` and `trivial` (default).
+
+ - `compose /logl`: Sets the minimum log level required to emit a log message
+   with `handler_log`. The default value is 2.
+
+ - `compose /rtsb`: Sets the maximum size of the RTS list (or packet buffer in
+   buffering mode). The The default value is 4096.
```
Comment (mvousden) [inline source/user_guide.md]: The The

File: source/user_guide.md [hunk 5]
```diff
@@ -666,35 +666,61 @@ configured staging directory (`Output/Composer` in the default configuration).
...
+
+ - `compose /buff`: Indicates that the Softswitch should be compiled in the
```
Comment (mvousden) [inline source/user_guide.md]: Refer to documentation XYZ for an explanation of buffering mode? (maybe a short explanation here would be nice too)

**Review by mvousden — APPROVED** [review]

File: source/user_guide.md (hunk 1 above)
Comment (heliosfa) [reply source/user_guide.md:694]: "clearing all data" is meant to be an aside, so the tense is consistent with "Deletes the composer"

File: source/user_guide.md (hunk 5 above)
Comment (heliosfa) [reply source/user_guide.md]: The short explanation would be quite convoluted. I have signposted to Vol III(B)

File: source/user_guide.md (hunk 4 above)
Comment (heliosfa) [reply source/user_guide.md]: 👍  serves me right for trying to use the editor on Github

File: source/user_guide.md (hunk 3 above)
Comment (heliosfa) [reply source/user_guide.md]: 👍

File: source/user_guide.md (hunk 2 above)
Comment (heliosfa) [reply source/user_guide.md]: 👍 serves me right for trying to use the editor on Github

**Pull request merged**